numba/tests/test_pycc.py

```diff
@@ -353,13 +353,52 @@ def run_python(args):
             """
         run_python(["-c", code])
 
+    def check_setup_nested_py(self, setup_py_file):
+        # Compute PYTHONPATH to ensure the child processes see this Numba
+        import numba
+        numba_path = os.path.abspath(os.path.dirname(
+                                     os.path.dirname(numba.__file__)))
+        env = dict(os.environ)
+        if env.get('PYTHONPATH', ''):
+            env['PYTHONPATH'] = numba_path + os.pathsep + env['PYTHONPATH']
+        else:
+            env['PYTHONPATH'] = numba_path
+
+        def run_python(args):
+            p = subprocess.Popen([sys.executable] + args,
```

Comment:
I am not sure that you need the full capability of `Popen` here. Maybe it will be easier and more legible if you use `run()` as described here: https://docs.python.org/3.6/library/subprocess.html#using-the-subprocess-module -- as of 0.48.0 the lowest Python version Numba supports is Python 3.6, so you don't need to worry about backwards compatibility.

Reply:
please compare with ` def check_setup_py` the only change is the addition of `nested.`